Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -73,6 +78,8 @@
 "compass-direction" = "Spoken description of compass direction, e.g. \"at 3 o'clock\" means \"to the right\", \"at 11 o'clock\" means \"slightly to the left\", etc. $1 is the hour.";
 "continue-reading-empty-description" = "Explore Wikipedia for more articles to read";
 "continue-reading-empty-title" = "No recently read articles";
+"dark-theme-display-name" = "Dark theme name presented to the user";
```

Comment:
Oh, can we name "dark-theme-display-name", "default-theme-display-name" and "sepia-theme-display-name" to start with "theme-" so they group together?